src/handlers/proposalsHandlers.ts

```diff
@@ -56,7 +62,175 @@ const getProposals = (
     });
 };
 
-const getProposal = (
+/**
```

Comment:
We haven't tended to use docblocks in this repo and part of that is that we try to make sure the names (and resulting intellisense) are self explanatory -- I think that applies here.

For instance if you rename the parameters to `shallowProposal` and `fieldValues` and if you name the function `decorateProposalWithFieldValues` I think that becomes auto-documenting and you can ditch the doc block.  (one benefit of this approach is we don't have to update two places if we make changes!)

Reply:
A noble goal. Now we merely have to agree on a name that communicates clearly. It sounds like `merge` is not the word you would use and I don't think `decorate` is quite right either. Perhaps `join`?

Reply:
merge is fine too, but I do like `join` since it alludes to the parallel DB operation -- FWIW I decorate is an allusion to the Decorator pattern -- https://en.wikipedia.org/wiki/Decorator_pattern -- which takes an object and dynamically adds (decorates) functionality to it.

Reply:
Yes, the old GoF. I tend to think of code as different from data, process as different from product, etc. In functional language communities, there need not be such a distinction. Then again, Design Patterns talked about OOP and mentions adding behavior, so that is why I think "decorator" is a little bit off: these are data.

Reply:
I used `join` but I am OK with going back to `merge`. For the reason above (a connotation of adding behavior, not data) I dislike `decorate`.